app/routes/__frontend/events/index.jsx

```diff
@@ -51,6 +51,23 @@ export default function EventsIndex() {
 						</p>
 					</div>
 
+					<h2>Recorded Events</h2>
+					<div className="mb-3 lead">
+						<p>
+							If you can't join our events live, we got you covered! Some of our
+							events, such as Lunch & Learns, are recorded and uploaded to our{' '}
+							<a href="https://www.youtube.com/channel/UCc0579aGEy7jTAgRglR4J0g">
+								YouTube channel
+							</a>
+							.
+						</p>
+						<p>
+							Please be patient if you can't find the events you're looking for.
+							Our volunteers are working their best to process and upload them
+							to our YouTube channel 😊. So remember to check it out!
+						</p>
```

Comment:
I appreciate the thought, but I don't think we need this paragraph.

Reply:
Gotcha! I've deleted this paragraph.
I wrote this because sometimes our members are asking particular events that are not uploaded yet to our YouTube channel 😁